packages/framework/store/src/store/doc/doc.ts

```diff
@@ -28,11 +29,17 @@ export class Doc {
     return this._blockCollection;
   }
 
+  get readonly$() {
+    return computed(() => {
```

Comment:
Same here

Reply:
After testing, I found that both computed signals are redundant.